src/emulator/modules/SceNgs/SceNgs.cpp

```diff
@@ -41,8 +43,10 @@ EXPORT(int, sceNgsPatchRemoveRouting) {
     return UNIMPLEMENTED();
 }
 
-EXPORT(int, sceNgsRackGetRequiredMemorySize) {
-    return UNIMPLEMENTED();
+EXPORT(int, sceNgsRackGetRequiredMemorySize, uint32_t hSysHandle, void *unk, uint32_t *size) {
```

Comment:
`hSysHandle` isn't in our style guide, change to `sys_handle` or something

Reply:
That's the original name of the arg (when added to vita-headers in the near future, it will use that arg naming)